admin/schedule-form-excludes.php

```diff
@@ -8,10 +8,10 @@
 <div class="hmbkp-exclude-settings">
 
 	<h3>
-		<?php _e( 'Currently Excluded', 'backupwordpress' ); ?>
+		<?php esc_html_e( 'Currently Excluded', 'backupwordpress' ); ?>
 	</h3>
 
-	<p><?php _e( 'We automatically detect and ignore common <abbr title="Version Control Systems">VCS</abbr> folders and other backup plugin folders.', 'backupwordpress' ); ?></p>
+	<p><?php esc_html_e( 'We automatically detect and ignore common Version Control Systems folders and other backup plugin folders.', 'backupwordpress' ); ?></p>
```

Comment:
I've got rid of `abbr` here, I don' think there is much for the benefit of having it - I'd prefer "don't make me think" approach when it's spelt straight away. Also makes escaping easier ;)